File: articles/evaluation-benchmarks-and-the-limits-of-ai-measurement/cpp/evaluation_metric.cpp
#include <iostream>

int main() {
    double tp = 42, tn = 38, fp = 7, fn = 13;
    double accuracy = (tp + tn) / (tp + tn + fp + fn);
    std::cout << "accuracy=" << accuracy << "\n";
    return 0;
}

File: articles/evaluation-benchmarks-and-the-limits-of-ai-measurement/java/EvaluationMetric.java
public class EvaluationMetric {
    public static void main(String[] args) {
        double tp = 42, tn = 38, fp = 7, fn = 13;
        double accuracy = (tp + tn) / (tp + tn + fp + fn);
        System.out.printf("accuracy=%.4f%n", accuracy);
    }
}
